### ISSUE TITLE
M15 R2 and RTX 3070

### ISSUE THREAD

**Issue by dongngm** [Shidell/OpenCaldera#11]
Hi,

Recently I bought a Zotac rtx 3070 to put into the AGA. Sadly when I turn it on, the card does not show up in Device Manager. I have confirmed that the AGA is still working fine by swapping with an old RX 460. I've also confirmed that the card is working fine in another system before buying it.
I try to run Caldera but it throws some errors.
I really don't know what to do next. Could you please shed some light on it? Thanks a lot

**Comment by Shidell** [comment]
@dongngm Zotac GPUs are notoriously bad for compatibility with the AGA for reasons unknown, suspected as a firmware issue (on the GPU itself.) I would recommend returning the Zotac and choosing a different manufacturer.

**Comment by dongngm** [comment]
@Shidell thank you so much. It's indeed the incompatibility with Zotac cards. I have tried another Zotac and it didn't work either. I switched to a Gigabyte and now all is fine although having some trouble with installing drivers. After many trials and errors this works best for me:
- uninstall drivers using DDU, disable automatic Windows driver installation (in DDU also)
- install the driver from Dell
- install the nvidia desktop driver
- this may not be necessary but I also disable the onboard Nvidia card

Hope it will help others
